ticker/src/main/java/com/robinhood/ticker/TickerColumnManager.java

```diff
@@ -52,25 +47,12 @@
     /**
      * @inheritDoc TickerView#setCharacterLists
      */
-    void setCharacterLists(String... characterLists) {
-        this.characterLists = new ArrayList<>(characterLists.length);
-        this.characterIndicesMaps = new ArrayList<>(characterLists.length);
+    void setCharacterLists(TickerCharacterList... characterLists) {
```

Comment:
since this used to take in a String... and each TickerCharacterList takes in a String, can we hide TickerCharacterList from the api, and just wrap them internally here?

Reply:
yeah i dont see a huge use-case for exposing this for allowing extensibility so ill change the public API to use string